### PR TITLE
1.8.9 train 167

### PR DESCRIPTION
**Pull request by adam-mesos**
:steam_locomotive: 167 for 1.8.9

#1651 navstar and minuteman bumps for 1k node cluster testing

### CODE REVIEW

**Comment by adam-mesos** [comment]
@mesosphere-mergebot bump-ee

**Comment by adam-mesos** [comment]
From @orsenthil - https://mesosphere.slack.com/archives/C0L6U3ECQ/p1497562074408667
> It seems that our upgrade test failed with 1.8.9 branch itself and it is unfortunately getting attention in 1651.
So, if we merge this, we won't necessarily be any worse than what's already in 1.8.9, but we'll need to a) do manual upgrade testing, or b) fix the test to work again.

**Comment by jgehrcke** [comment]
`TestUpgrade.test_mesos_task_state_remains_consistent` failed in https://teamcity.mesosphere.io/viewLog.html?buildId=684493. Restarted it.

**Comment by GoelDeepak** [comment]
@jgehrcke restart will not help, I tried that multiple times before.

File: packages/minuteman/buildinfo.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "minuteman": {
       "kind": "git",
       "git": "https://github.com/dcos/minuteman.git",
-      "ref": "1c45dab611b13cb3a491a2b41f2dc7d36ffde7b2",
+      "ref": "5fff0caf44d15f734cfb961338b04aec1dc3424f",
       "ref_origin": "dcos/1.8"
```
Comment (klueska) [inline packages/minuteman/buildinfo.json]: I don't see the equivalent of this commit anywhere on the `dcos/1.8` branch for `navstar`:
```
commit 4079543987fd7b1c0b03b97110e2a97a883b8dcd
Merge: 68c634f af5a550
Author: Deepak Goel <dgoel@mesosphere.com>
Date:   Thu Mar 23 01:21:52 2017 -0700

    Merge pull request #58 from dcos/fix-dcos-oss-804
    
    log message to include pid, module, function, line number
```

Amy I missing something?

File: packages/navstar/buildinfo.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "navstar": {
       "kind": "git",
       "git": "https://github.com/dcos/navstar.git",
-      "ref": "27bbade72cfedb6a063860adfbbd5ba551ccafad",
+      "ref": "68997ac64fb6c6fbf1906323be565ace18756622",
       "ref_origin": "dcos/1.8"
```
Comment (klueska) [inline packages/navstar/buildinfo.json]: I don't see the equivalent of this commit anywhere on the `dcos/1.8` branch for `spartan`:
```
commit 927050af455802057d920d2adb03a84eefcd27e7
Merge: 553d0dc 0b5243b
Author: Deepak Goel <dgoel@mesosphere.com>
Date:   Thu Mar 23 01:03:24 2017 -0700

    Merge pull request #33 from dcos/fix-dcos-oss-804
    
    log message to include pid, module, function, line number
```

Am I missing something?

**Comment by klueska** [comment]
Actually, I'm more confused in general. I thought we wanted to backport the fix for this bug:
https://jira.mesosphere.com/browse/DCOS_OSS-796

Which links to the following PR that updates *spartan* and navstar:
https://github.com/dcos/dcos/pull/1391

The current PR updates *minuteman* and navstar.

**Comment by GoelDeepak** [comment]
@klueska You should see the referenced commits now. Also, there are multiple fixes that go in a single DCOS PR. Further, minuteman repo has been merged to navstar in 1.9

**Comment by mellenburg** [comment]
For this upgrade test to pass, some manual intervention must be done.
First, use this CI job, not the one that is automatically run: https://teamcity.mesosphere.io/viewType.html?buildTypeId=ClosedSource_Dcos_IntegrationTests_CloudIntegrationTests_DcosOssUpgradeTestOnPre
Second, you need to specifically set the starting installer as described here: https://wiki.mesosphere.com/display/DCOS/Major+Version+Differences

**Comment by mellenburg** [comment]
To be perfectly explicit, when you use the above (legacy) CI job, you must set env.STABLE_INSTALLER_URL to https://downloads.dcos.io/dcos/stable/commit/602edc1b4da9364297d166d4857fc8ed7b0b65ca/dcos_generate_config.sh for this to pass

**Comment by orsenthil** [comment]
I have triggered the proper upgrade integration test already: https://teamcity.mesosphere.io/viewLog.html?buildId=685366

**Comment by orsenthil** [comment]
This is the JIRA that captures the the upgrade integration test failure: https://jira.mesosphere.com/browse/DCOS-16285

**Comment by orsenthil** [comment]
We are at-least no longer held up upgrade test failures. I am closing the JIRA that I created today morning.

**Comment by adam-mesos** [comment]
From slack: https://mesosphere.slack.com/archives/C0L6U3ECQ/p1497648063968239?thread_ts=1497565585.224012&cid=C0L6U3ECQ
> dgoel spartan bump isn't required for 1K node cluster fix

Also, with the updated component changelogs, I can verify that we are indeed bumping navstar+minuteman to include a lashup with the large object fix.

I'm going to merge.

**Pull request merged**